### ISSUE TITLE
[KKBP 7.0.0] Automatic atlas generation for child meshes does not import atlas file

### ISSUE THREAD

**Issue by macdoniel** [FlailingFog/KK-Blender-Porter-Pack#598]
Hi, thanks for all your work! I've been experimenting with the 7.0.0 RC, and would like some clarification on what might be a bug, or lack of understanding.

I'm trying to export a mesh `Outfit 00` , where I separated out a mesh `Hat`  manually.

![image](https://github.com/user-attachments/assets/29b31ffc-1421-4651-b62d-145fcf46517e)

Everything looks fine in the pre-finalized model. When I create the atlas, the files for "Hat atlas light" and "Hat atlas dark" are created successfully. However, the model does not seem to pick them up automatically, and instead uses the atlas for the parent mesh:

![image](https://github.com/user-attachments/assets/011a93c0-fff9-4018-9546-65fdb6966e45)

As shown, this happens on that hat, and the glasses, which is another sub mesh. The UVs seem to be configured for the "Hat atlas" file, but need to be translated.

![image](https://github.com/user-attachments/assets/39737b72-1d0e-477e-b71e-e530abc13171)

Is this perhaps an issue with the model, the plugin, or a misunderstanding? Thanks in advance

KK card:

![Koikatu_F_20240831033638142_Kirisame Marisa](https://github.com/user-attachments/assets/23d1cb9d-209e-4f2e-9158-5d56b1e41907)

**Comment by FlailingFog** [comment]
Hmm... I was able to confirm this, but I would have to look into why it's happening, that's pretty weird. I do think it's the plugin though. 

For now, you can probably separate everything after the atlas is generated. I think that'd still work.

**Comment by daniel-koudouna** [comment]
Thanks for the confirmation! Another thing that happens is that the atlas materials are not combined, e.g 30 hair accessories still have 30 atlas materials. They can be easily merged manually, but it could be worth looking into if it's unintended.

**Comment by macdoniel** [comment]
Thanks for the confirmation! Another thing that happens is that the atlas materials are not combined, e.g 30 hair accessories still have 30 atlas materials. They can be easily merged manually, but it could be worth looking into if it's unintended.

**Comment by FlailingFog** [comment]
Yes, keeping the multiple atlas slots was intentional. I plan to keep it that way and let the user decide if they want to merge them or not (since it's only a few clicks to merge them anyway).
